go/pkg/client/capabilities.go

```diff
@@ -46,7 +46,7 @@ func (c *Client) CheckCapabilities(ctx context.Context) (err error) {
 		}
 		for _, compressor := range c.serverCaps.CacheCapabilities.SupportedBatchUpdateCompressors {
 			if compressor == repb.Compressor_ZSTD {
-				c.batchCompression = true
+				c.BatchCompression = true
```

Comment:
It looks like this was made public to be accessible from the test package? If so, the existing patter is define public configs as client options, i.e. https://github.com/bazelbuild/remote-apis-sdks/blob/2ab761126c7ac5a597f5a93acd53acd2c028e97d/go/pkg/client/client.go#L270

Reply:
Thanks for the context too, good point. Done!